pkg/view/component/link.go

```diff
@@ -19,8 +19,9 @@ var _ Component = &Link{}
 
 // LinkConfig is the contents of Link
 type LinkConfig struct {
-	Text string `json:"value"`
-	Ref  string `json:"ref"`
+	Text    string    `json:"value"`
+	Ref     string    `json:"ref"`
+	Content Component `json:"content,omitempty"`
```

Comment:
I'd like to see a comment and/or documentation around priority: if you provide both Content and Text, you're Content your Text will get swallowed up.